api/room/schema_query.py

```diff
@@ -324,8 +333,21 @@ def resolve_analytics_for_daily_room_events(self, info,
                     events=daily_events
                 )
             )
-
-        return all_days_events
+        if page and per_page:
+            paginated_results = ListPaginate(iterable=all_days_events, per_page=per_page, page=page)  # noqa: E501
```

Comment:
Hello @verenceLola , Is it possible to get rid of this noqa?

Reply:
This has been worked on. Thanks for the observation @nakatuddesuzan